openhands/agenthub/tom_codeact_agent/tom_codeact_agent.py

```diff
@@ -88,6 +89,24 @@ def __init__(self, config: AgentConfig, llm_registry: LLMRegistry) -> None:
         self._last_processed_user_message_id: Optional[int] = None
         self._skip_next_tom_analysis: bool = False
 
+        # Track ToM agent initialization
+        try:
```

Comment:
If we don't have session id here, i would suggest removing the track mechanism here, we could do a first time session tracking at the beginning of step function (only when we are using CLI)